client/src/App.js

```diff
@@ -3,6 +3,7 @@ import React, { Component } from 'react';
 import SavedList from './Movies/SavedList';
 import MovieList from './Movies/MovieList';
 import Movie from './Movies/Movie';
+import {BrowserRouter as Router, Route, Link, NavLink} from "react-router-dom";
```

Comment:
its not necessary to import all of these things here,  only what you need in the file.  it is however, not hurting anything.